test/minitest_config.rb

```diff
@@ -9,3 +9,27 @@
 require "minitest/moar"
 
 require "marlowe"
+
+module NormalizeRackResponseHeaders
+  private
+
+  def has_header?(key)
+    if Rack.release.start_with?("1")
```

Comment:
(no action required) This can be a problem if rack makes it to release "10.0.0"

Reply:
Fixed by switching to `"1."`, even though the development pace of Rack suggests that Rack 10 is ~15–20 years away. 😆 Policies could change, so it's a nice and easy fix.

I also changed to using a constant (`RackV1 = Rack.release.start_with?("1.")`) and using that rather than calling a function each time.